templates/wipe-disk.sh.j2

```diff
@@ -0,0 +1,29 @@
+{% if ceph_enabled %}
+#!/bin/bash
+if [[ `hostname` = *"ceph"* ]]
+then
+  sudo yum install gdisk -y
```

Comment:
Will "yum install" work on nodes. Usually I see the node not having any repos.

Reply:
Ah, I missed it